pkg/archive/archive.go

```diff
@@ -217,6 +223,10 @@ func CompressStream(dest io.Writer, compression Compression) (io.WriteCloser, er
 		gzWriter := gzip.NewWriter(dest)
 		writeBufWrapper := p.NewWriteCloserWrapper(buf, gzWriter)
 		return writeBufWrapper, nil
+	case Zstd:
+		zStdWriter := gzip.NewWriter(dest)
+		writeBufWrapper := p.NewWriteCloserWrapper(buf, zStdWriter)
+		return writeBufWrapper, nil
```

Comment:
Shouldn't this be returning a `*zstd.Writer`?

Reply:
ouch yes sure, will fix it